script/Arcueid/Day 7/07_00_ARC07_3_1_3.txt

```diff
@@ -404,6 +408,7 @@ A crimson stain that dripped from the moon and fell upon the earth.
 // 世界を持_った_ is a little weird tense-wise, are they no longer this way?
 // Possessive correlation reversed here for EN (they are of a world, they do not hold the world)
 // たち on 猛毒 used as basis for 'they', instead of being subjectless like the prev sentence.
+// Group PR Mark on this sentence
```

Comment:
Another lore one @yk1512

Reply:
I personally agree with the current interpretation